libs/api/domains/driving-license-book/src/dto/activeBookId.input.ts

```diff
@@ -0,0 +1,13 @@
+import { IsString } from 'class-validator'
+import { Field, InputType } from '@nestjs/graphql'
+
+@InputType()
+export class ActiveBookIdInput {
+  @Field({ nullable: true })
+  @IsString()
+  ssn!: string
```

Comment:
we use nationalId in island.is ... btw. the class-validators don't do anything here... can be removed.